cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -87,38 +103,84 @@ executions {
     }
 }
 
-config.policies.each{ policySettings ->
-    def cron = policySettings[ 0 ] ? policySettings[ 0 ] as String : ["0 0 5 ? * 1"]
-    def repos = policySettings[ 1 ] ? policySettings[ 1 ] as String[] : ["__none__"]
-    def months = policySettings[ 2 ] ? policySettings[ 2 ] as int : 6
-    def paceTimeMS = policySettings[ 3 ] ? policySettings[ 3 ] as int : 0
-    def dryRun = policySettings[ 4 ] ? policySettings[ 4 ] as Boolean : false
-    def disablePropertiesSupport = policySettings[ 5 ] ? policySettings[ 5 ] as Boolean : false
-    def keepArtifacts = policySettings[ 6 ] ? policySettings[ 6 ] as Boolean : false
-    def keepArtifactsRegex = policySettings[ 7 ] ? policySettings[ 7 ] as Pattern : regex
+def deprecatedConfigFile = new File(ctx.artifactoryHome.etcDir, PROPERTIES_FILE_PATH)
+def configFile = new File(ctx.artifactoryHome.etcDir, CONFIG_FILE_PATH)
+
+if ( deprecatedConfigFile.exists() ) {
+
+    if ( !configFile.exists() ) {
+        def config = new ConfigSlurper().parse(deprecatedConfigFile.toURL())
+        log.info "Schedule job policy list: $config.policies"
+
+        def count=1
+        config.policies.each{ policySettings ->
+            def cron = policySettings[ 0 ] ? policySettings[ 0 ] as String : ["0 0 5 ? * 1"]
+            def repos = policySettings[ 1 ] ? policySettings[ 1 ] as String[] : ["__none__"]
+            def months = policySettings[ 2 ] ? policySettings[ 2 ] as int : 6
+            def paceTimeMS = policySettings[ 3 ] ? policySettings[ 3 ] as int : 0
+            def dryRun = policySettings[ 4 ] ? policySettings[ 4 ] as Boolean : false
+            def disablePropertiesSupport = policySettings[ 5 ] ? policySettings[ 5 ] as Boolean : false
+            def keepArtifacts = policySettings[ 6 ] ? policySettings[ 6 ] as Boolean : false
+            def keepArtifactsRegex = policySettings[ 7 ] ? policySettings[ 7 ] as Pattern : regex
 
+            jobs {
+                "scheduledCleanup_$count"(cron: cron) {
+                    log.info "Policy settings for scheduled run at($cron): repo list($repos), timeUnit(month), timeInterval($months), paceTimeMS($paceTimeMS) dryrun($dryRun) disablePropertiesSupport($disablePropertiesSupport)"
```

Comment:
`keepArtifacts` and `keepArtifactsRegex` should be dumped as well in this log message